src/systems/detachable_joint/DetachableJoint.cc

```diff
@@ -225,54 +314,35 @@ void DetachableJoint::PreUpdate(
     if (!this->attachRequested){
       return;
     }
-    // Look for the child model and link
-    Entity modelEntity{kNullEntity};
 
-    if ("__model__" == this->childModelName)
+    if (childLinkEntity == kNullEntity || !_ecm.HasEntity(childLinkEntity))
```

Comment:
```suggestion
    if (this->childLinkEntity == kNullEntity || !_ecm.HasEntity(this->childLinkEntity))
```
to be consistent with style (sorry should have suggested this in the first place)

Reply:
Nah it's fine. I should be more careful also :stuck_out_tongue: [6fa7157](https://github.com/gazebosim/gz-sim/pull/1097/commits/6fa7157a7beaa8735ecbde4dcc037934e3ca389c)